src/server/routes/needs.ts

```diff
@@ -0,0 +1,43 @@
+import express from 'express'
+import type { Response, Request } from 'express'
+import type { fetchBody } from '../../app/types/types'
+import { getBoards } from '../../utils/db'
+
+const needs = express.Router()
+
+needs.post('/', async (req: Request, res: Response) => {
+  const { boardName: name, topicId, payload: newNeed }: fetchBody = req.body
+
+  const boards = await getBoards()
+  const msg = await boards.updateOne(
```

Comment:
msg is not very good explainign what is happening here. I saw you using it everywhere else as well, maybe for next time, keep in mind :)